graphgrid_sdk/ggcore/api.py

```diff
@@ -223,8 +264,89 @@ def endpoint(self):
         def http_method(self) -> HttpMethod:
             return HttpMethod.POST
 
-        def handler(self, sdk_response: SdkServiceResponse):
-            return sdk_response
+        def handler(self, generic_response: GenericResponse):
+            return PromoteModelResponse(generic_response)
+
+    @dataclass
+    class GetJobResultsApi(AbstractApi):
+        """Define GetJobResultsApi api."""
+        _dag_id: str
+        _dag_run_id: str
+
+        def __init__(self, dag_id: str, dag_run_id: str):
+            self._dag_id = dag_id
+            self._dag_run_id = dag_run_id
+
+        def api_base(self) -> str:
+            return NLP
+
+        def endpoint(self):
+            return f"status/{self._dag_id}/{self._dag_run_id}"
+
+        def body(self):
+            return {
+                "dag_id": self._dag_id,
+                "dag_run_id": self._dag_run_id
+            }
```

Comment:
Is this necessary? The `dag_id` and `dag_run_id` are being passed via path variables in the endpoint, and no body is expected unless I'm mistaken
(same for job status)

Reply:
good catch, I'll fix this.